documentation/BUILDBOT.rst

```diff
@@ -12,7 +12,7 @@ http://build.cmi.kent.edu:8010/
 Modifying the configuration
 ---------------------------
 
-The buildbot configuration file is hosted in a Git repository. To clone the
+The Buildbot configuration file is hosted in a Git repository. To clone the
```

Comment:
We could delete this file. Shall I open an issue or do you want to do it as part of this pull request.

Reply:
Deleted.